packages/esm-patient-list-app/src/ui-components/overflow-menu.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+
+interface CustomOverflowMenuComponentProps {
+  menuTitle: React.ReactNode;
+}
+
+const CustomOverflowMenuComponent: React.FC<CustomOverflowMenuComponentProps> = ({ menuTitle, children }) => {
+  const [showMenu, setShowMenu] = useState(false);
+  const wrapperRef = useRef(null);
+  const toggleShowMenu = useCallback(() => setShowMenu((state) => !state), []);
+
+  useEffect(() => {
+    /**
+     * Toggle showMenu if clicked on outside of element
+     */
+    function handleClickOutside(event: MouseEvent) {
+      if (wrapperRef.current && !wrapperRef.current.contains(event.target)) {
+        setShowMenu(false);
+      }
+    }
+
+    // Bind the event listener
+    document.addEventListener('mousedown', handleClickOutside);
+    return () => {
+      // Unbind the event listener on clean up
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, [wrapperRef]);
+
+  return (
+    <div
+      data-overflow-menu
+      className="bx--overflow-menu"
+      style={{
+        width: 'auto',
+        height: 'auto',
+        marginBottom: '-1.5rem',
+        marginTop: '-1.25rem',
+      }}
+      ref={wrapperRef}>
+      <button
+        className={`bx--overflow-menu__trigger ${showMenu && 'bx--overflow-menu--open'}`}
+        aria-haspopup="true"
+        aria-expanded={showMenu}
+        id="custom-actions-overflow-menu-trigger"
+        aria-controls="custom-actions-overflow-menu"
+        onClick={toggleShowMenu}
+        style={{
+          width: 'auto',
+          height: 'auto',
+          padding: '1rem',
+          color: '#0f62fe',
+          outline: '2rem solid transparent',
+          boxShadow: showMenu ? '0 2px 6px 0 rgb(0 0 0 / 30%)' : 'none',
```

Comment:
Would be awesome if all the styles in this file were moved to an SCSS file. For the conditional ones (e.g. `boxShadow: showMenu ? '0 2px 6px 0 rgb(0 0 0 / 30%)' : 'none'` here) you can either use conditional class names or leave them as inline styles like it is done here.

Reply:
Sure!

Reply:
Ping. Looks like the blue is blue-60 from https://app.zeplin.io/styleguide/60d5ecb9efdcd81256117e7d/colors